docs/CoC.md

```diff
@@ -1,20 +1,31 @@
-- **Artist** - This role is for anyone who makes artwork. To get this role you must contact a moderator and provide a link to your portfolio. While there is no quality control for this role, you must have some history in making artwork. Verification may be required.
-- **Community rep** - This role is for members that we consider the best representatives of what we like to see in members of the community.
-- **Modpack Maker** - This role is for Modpack developers. There are no size/download count requirements, however you will be required to provide some verification in form of screenshot or any other form the moderator sees fitting. (Editing the description of the mod on CurseForge with a message for the staff is prefered but other options are avalable)
-- **MMD Public Server Players** - This role is for anyone who plays on MMD's public server. Should you want to be pinged with information about server and updates to the modpack please mention`@MMD Public Server Admins` and request the role be added to your account.  
+- **Artist/Modeler** - This role is for anyone who makes artwork. To get this role you must contact a moderator and provide a link to your portfolio. While there is no quality control for this role, you must have some history in making artwork. Verification may be required.
+- **Community Rep** - This role is for members that we consider the best representatives of what we like to see in members of the community.
+- **Modpack Maker** - This role is for Modpack developers. There are no size/download count requirements, however you will be required to provide some verification in form of screenshot or any other form the moderator sees fitting. (Editing the description of the mod on CurseForge with a message for the staff is preferred but other options are available)
+- **Translator** - This role is for anyone who translates mods to other languages. 
+- **Public Server Player** - This role is for anyone who plays on MMD's public server. Should you want to be pinged with information about server and updates to the modpack, please mention `@Public Server Admin` and request the role be added to your account.  
+
+### Other Roles
+These roles are given at the discretion of the MMD staff when certain conditions are met. These roles should not be requested in `#role-requests`.
+
+- **Bot** - This role is for Discord bots that operate on the server, under the guidance of the maintainers.
+- **Bot Maintainer** - This role is for the people that maintain the Discord bots on the server.
+- **Retired Staff** - This role is for staff who retire from their staff position and powers.
+- **SpookyJam 2019** - This role is for anyone who submitted an entry for SpookyJam 2019. 
```

Comment:
You are missing the 2020 spooky role :p other than that the PR looks great.

Reply:
I based that other roles list on the currently visible roles that wasn't available in the community list (mostly by looking at your role list on Discord 😄). I'll add the role.
Also, I assume Event Team is a temporary role for the duration of the SpookyJam? Or will it be used for e.g. WinterJam too? If it stays, I will also include it into the section.

Reply:
That one I am unsure of but Dark might know, that role is above my pay grade lol

Reply:
I'll ask Darkhax once they're online, but I'll refrain from adding the role until it's confirmed that Event Team role is not only for SpookyJam (and that's a likely possibility).

Reply:
Added Event Team role as Darkhax has said that the role is to be used in the future for all events.
> [**sciwhiz12**: btw Dark, since you're here, asking for my PR to MMD-Site: is the Event Team role on MMD temporary for SpookyJam, or will be used again for WinterJam?](https://discord.com/channels/752944675425353768/752993307348697264/771798163768999976)
[**Darkhax**: Yeah, I believe we're going to use it for all our events in the future.](https://discord.com/channels/752944675425353768/752993307348697264/771798546604621844)